state-chain/primitives/src/chains/assets.rs

```diff
@@ -389,6 +389,13 @@ macro_rules! assets {
 					})
 				}
 			}
+
+			impl scale_info::prelude::ops::Add<AssetMap<crate::AssetAmount>> for AssetMap<crate::AssetAmount> {
```

Comment:
nit: it's a bit strange to import this via scale_info... 

```suggestion
			impl core::ops::Add<AssetMap<crate::AssetAmount>> for AssetMap<crate::AssetAmount> {
```